crates/types/src/tx.rs

```diff
@@ -34,8 +36,8 @@ pub enum Message {
     ///
     /// For app-level config, setting a value to `Null` means to delete it.
```

Comment:
Remove this comment

Reply:
What's wrong with this comment? If you wish no delete a key in the app config, set it to `Null`. If you don't wish to change it, don't include it in `app_updates` map.